base_tier_validation/models/tier_definition.py

```diff
@@ -8,11 +8,17 @@ class TierDefinition(models.Model):
     _name = "tier.definition"
     _rec_name = "model_id"
```

Comment:
Since now we have a `name`, maybe we should use it (`name`) instead of `model_id` for `_rec_name`?